### PR TITLE
Link from Policies concept to NetworkPolicy page

### PR DESCRIPTION
**Pull request by sftim**
Split out from https://github.com/kubernetes/website/pull/30817

Link from [Policies](https://kubernetes.io/docs/concepts/policy/) concept to [Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) page. [[preview](https://deploy-preview-36670--kubernetes-io-main-staging.netlify.app/docs/concepts/policy/)]

### CODE REVIEW

**Comment by sftim** [comment]
Helps with https://github.com/kubernetes/website/issues/14770

File: content/en/docs/concepts/policy/_index.md [hunk 1]
```diff
@@ -4,3 +4,8 @@ weight: 90
 description: >
   Policies you can configure that apply to groups of resources.
 ---
+
+{{< note >}}
+See [Network Policies](/docs/concepts/services-networking/network-policies/)
+for documentation about NetworkPolicy in Kubernetes.
```
Comment (kbhawkey) [inline content/en/docs/concepts/policy/_index.md:10]: Hi @sftim. Is there a good reason to include a link to the Network Policies section from this index page? Why does this information need to be in a note?

Comment (sftim) [reply content/en/docs/concepts/policy/_index.md:10]: If someone is looking for network policies, they might look for it in the Policies section, but it's actually in the Networking section. This note is to help people who looked in Policies find the thing that they are looking for.

Maybe we have a different mechanism to add it into the navigation itself; if we do, I don't know about that other section.

**Pull request merged**